project_task_category/models/project_task_category.py

```diff
@@ -11,4 +11,5 @@ class ProjectTask(models.Model):
     categ_id = fields.Many2one(
         'project.category',
         'Category',
+        domain=[('code', '=', 'categ_id')]
```

Comment:
> victormartinelicocorp I slept on this, and I think that adding a flag to the Category model can do the trick. It does have have a few details to get right that I can share with you. Do you prefer to go this way?

@dreispt Actually I am ok with both solutions my latest commit I follow the way of create new model with inheritance.

If I have to choose I feel a little more right to have a separated DB tables so rather keep my latest commit: https://github.com/OCA/project/pull/239/commits/727c42e301719efbf945083dd51fe158cb466bd4

Please let me know if need to change anything and I'll do the necessary.

cc @elicoidal